utils/scheduler.go

```diff
@@ -0,0 +1,38 @@
+package utils
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"../actions"
+	"../auth"
+	"github.com/robfig/cron"
+)
+
+func Schedule(client *http.Client, tokenResponse auth.AccessTokenResponse, backup actions.Backup) *cron.Cron {
```

Comment:
You've probably noticed a pattern emerging here, we'll refactor to use an ApiClient struct instead. Then we'd pass it into this function: `Schedule(apiClient, backup)`